pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -1,15 +1,20 @@
 from argparse import ArgumentParser
+from typing import Any, Optional
 
 import pytorch_lightning as pl
 import torch
 
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
 from pl_bolts.utils.warnings import warn_missing_pkg
 
-try:
-    from torchvision.models.detection import faster_rcnn, fasterrcnn_resnet50_fpn
+if _TORCHVISION_AVAILABLE:
+    from torchvision.models.detection.faster_rcnn import FasterRCNN as torchvision_FasterRCNN
+    from torchvision.models.detection.faster_rcnn import fasterrcnn_resnet50_fpn, FastRCNNPredictor
     from torchvision.ops import box_iou
-except ModuleNotFoundError:
-    warn_missing_pkg('torchvision')  # pragma: no-cover
+
+    from pl_bolts.models.detection.faster_rcnn import create_fasterrcnn_backbone
```

Comment:
I think we should place `from pl_bolts.models.detection.faster_rcnn import create_fasterrcnn_backbone` outside `if/else` since it should be possible to import `create_fasterrcnn_backbone` even when torchvision is not available.

Reply:
I don't think so it is possible to import / use `create_faster_rcnn_backbone` without torchvision, as it uses torchvision models and `resnet_fpn_bakcbone` from torchvision.